hack/lint-dependencies.sh

```diff
@@ -47,6 +47,19 @@ rc=0
 # The array and map in `unwanted-dependencies.json` are in alphabetical order.
 go run k8s.io/kubernetes/cmd/dependencyverifier "${KUBE_ROOT}/hack/unwanted-dependencies.json"
 
+k8s_module_regex="k8s[.]io/(kubernetes"
```

Comment:
we bothered to escape the `.` here but we aren't for the repos below, should we fix that?

Reply:
which repos below? list_staging_repos only gives the last path segment, none of which contain `.`